tests/subsys/fs/nffs_fs_api/basic/CMakeLists.txt

```diff
@@ -30,5 +30,9 @@ endif()
 
 
 zephyr_include_directories(../common)
+if(BOARD STREQUAL qemu_x86)
+FILE(GLOB app_sources ../common/*.c src/*.c ../ram_backend/*.c)
+else()
 FILE(GLOB app_sources ../common/*.c src/*.c)
+endif()
```

Comment:
Instead of this, you want to do

```
if(CONFIG_BOARD_QEMU_X86)
list(APPEND app_sources ../ram_backend/test_ram_backend.c)
endif()
```

Reply:
: D that I exactly what would to ask about